client/components/Dashboard.tsx

```diff
@@ -1,8 +1,8 @@
 import React, { useState, useEffect } from 'react';
```

Comment:
This `Queries` component is using serif while the chart uses sans-serif font. Let's pick one and standardize

Reply:
Ok! Not sure if you had time to read my reply for this issue, but will be fixed! :)